pkg/database/regex.go

```diff
@@ -0,0 +1,36 @@
+package database
+
+import (
+	"regexp"
+
+	lru "github.com/hashicorp/golang-lru"
+)
+
+const regexCacheSize = 10
```

Comment:
add a comment for the size? is it in MB? (the docs in the library are not mentioning anything)

Reply:
I'm flexible on the actual number used. I chose a small number because it's most likely use is for a single query - this function gets called for every row in the (filtered) results. It's likely to only need no more than 1 or 2 in any given query. After that point, it's just sitting in the cache and is unlikely to be used again. I think increasing the number is probably going to give negligible performance improvements at the cost of more memory usage. I have no idea how much memory a `regexp.Regexp` instance takes up.

Reply:
I've increased the cache size to 4096 and didn't see a change in memory usage (unless it was under 5MB), so there may be another issue somewhere...

Reply:
I just added a print when the cache is hit and it seems to me that it is like withoutpants mentioned above it is only once per query ( size =1 )
I did the below change(just for testing)and left it running some more time and it doesnt seem to get more than that

```diff
diff --git a/pkg/database/regex.go b/pkg/database/regex.go
index dc7b5feb..cdce9766 100644
--- a/pkg/database/regex.go
+++ b/pkg/database/regex.go
@@ -1,6 +1,7 @@
 package database
 
 import (
+       "fmt"
        "regexp"
 
        lru "github.com/hashicorp/golang-lru"
@@ -12,7 +13,7 @@ import (
 // results. It's likely to only need no more than 1 or 2 in any given query.
 // After that point, it's just sitting in the cache and is unlikely to be used
 // again.
-const regexCacheSize = 10
+const regexCacheSize = 100
 
 var regexCache *lru.Cache
 
@@ -36,6 +37,10 @@ func regexFn(re, s string) (bool, error) {
                regexCache.Add(re, compiled)
        } else {
                compiled = entry.(*regexp.Regexp)
+               length := regexCache.Len()
+               if length > 1 {
+                       fmt.Printf("cache usage %d", length)
+               }
        }
 
        return compiled.MatchString(s), nil
```